.github/workflows/gemini-review.yml

```
@@ -0,0 +1,122 @@
+          gcp_workload_identity_provider: '${{ vars.GCP_WIF_PROVIDER }}'
+          gemini_api_key: '${{ secrets.GEMINI_API_KEY }}'
+          gemini_cli_version: '${{ vars.GEMINI_CLI_VERSION }}'
+          gemini_debug: '${{ fromJSON(vars.GEMINI_DEBUG || vars.ACTIONS_STEP_DEBUG || false) }}'
+          gemini_model: '${{ vars.GEMINI_MODEL }}'
+          google_api_key: '${{ secrets.GOOGLE_API_KEY }}'
+          use_gemini_code_assist: '${{ vars.GOOGLE_GENAI_USE_GCA }}'
+          use_vertex_ai: '${{ vars.GOOGLE_GENAI_USE_VERTEXAI }}'
+          upload_artifacts: '${{ vars.UPLOAD_ARTIFACTS }}'
+          workflow_name: 'gemini-review'
+          # Explicitly set the PR number to handle `issue_comment` triggers (which GitHub treats as issues, not PRs)
+          github_pr_number: '${{ github.event.pull_request.number }}'
```

actionlint (1.7.12)

[error] input "github_pr_number" is not defined in action "google-github-actions/run-gemini-cli@v0". available inputs are "extensions", "gcp_access_token_scopes", "gcp_location", "gcp_project_id", "gcp_service_account", "gcp_token_format", "gcp_workload_identity_provider", "gemini_api_key", "gemini_cli_version", "gemini_debug", "gemini_model", "google_api_key", "prompt", "settings", "upload_artifacts", "use_gemini_code_assist", "use_pnpm", "use_vertex_ai", "workflow_name"

(action)